### DISCUSSION TITLE
Not able to cancel the commit inside lunarvim lazygit windown

### DISCUSSION THREAD

**Discussion by ksajan** [LunarVim/LunarVim#4504] [General]
1. Press <Prefix>gg
2. Press Space to stage the file
3. Press c to start the commit step
4. Now press q the commit box doesn't get canceled

When normally opened this works if I let's say have opened the commit box accidentally or for whatever reason I want to cancel, I cannot do so.
Can we toggle this feature on like some keybinding missing or needs to enabled/changed?

**Comment by olrtg** [comment]
Did you try pressing `Esc` to close the commit box?

**Reply by ksajan** [reply]
I did but it doesn't do anything.

**Reply by ksajan** [reply]
Event CTRL-o to choose cancel also doen't cancel the commit box
